chapter_multilayer-perceptrons/underfit-overfit_vn.md

```diff
@@ -310,21 +315,26 @@ If we overfit our training data, there is always the evaluation on test data to
 But if we overfit the test data, how would we ever know?
 -->
 
-*dịch đoạn phía trên*
+Về nguyên tắc là không nên chạm vào tập kiểm tra cho đến khi chọn được tất cả các siêu tham số.
+Ta đã sử dụng dữ liệu kiểm tra trong quy trình lựa chọn mô hình, rủi ro là ta có thể quá khớp dữ liệu kiểm tra.
```

Comment:
```suggestion
Nếu sử dụng dữ liệu kiểm tra trong quá trình lựa chọn mô hình, có một rủi ro là ta có thể quá khớp dữ liệu kiểm tra.
```

Tác giả rất thích dùng cụm `were we to do`. Cụm này lạ với mình, nhưng vừa check google xong thì nó đơn giản là "if we ..". Đây là câu điều kiện loại 2.

Reply:
em đã ghi chú lại. thanks anh.